cmd/gyee/configcmd.go

```diff
@@ -23,34 +23,32 @@ package main
 import (
 	"fmt"
 	"errors"
+
 	"github.com/urfave/cli"
 	"github.com/yeeco/gyee/config"
 )
 
 var (
 	configCommand = cli.Command{
-		Name:     "config",
-		Usage:    "Manage config",
-		Category: "CONFIG COMMANDS",
-		Description: `
```

Comment:
leading newline here was shown in help text, no need to format

Reply:
ok